apps/shell/app.js

```diff
@@ -5,13 +5,24 @@ var WebSocket = require('ws');
 var express   = require('express');
 var pty       = require('node-pty');
 var hbs       = require('hbs');
+var util      = require('util')
 var dotenv    = require('dotenv');
+var os        = require('os');
+var path      = require('path');
+var url       = require('url');
+var termSchemes   = require('term-schemes');
 var port = 3000;
 var uuidv4 = require('uuid/v4');
 
 //regular expression to find uuid in url
 const regexPathMatch = /[a-f0-9]{8}-?[a-f0-9]{4}-?4[a-f0-9]{3}-?[89ab][a-f0-9]{3}-?[a-f0-9]{12}/i;
+const regexFileMatch = /^.*\.(js|itermcolors|colorscheme|colors|terminal|config|config_0|theme|xrdb|xresources|txt)$/gmi;
+const xdg_config_dir = (process.env["XDG_CONFIG_DIR"] || path.join(os.homedir(), ".config"));
```

Comment:
I was mistaken. The env var is actually `XDG_CONFIG_HOME` which defaults to ~/.config